test/stat_tracker_test.rb

```diff
@@ -17,45 +18,100 @@ def setup
   end
 
   def test_it_exists
-    game_path = './data/games.csv'
-    team_path = './data/teams.csv'
-    game_teams_path = './data/game_teams.csv'
-
-      locations = {
-        games: game_path,
-        teams: team_path,
-        game_teams: game_teams_path
-      }
-    stat_tracker = StatTracker.from_csv(locations)
-    assert_instance_of StatTracker, stat_tracker
+    assert_instance_of StatTracker, @stat_tracker
```

Comment:
Also for taking out all the nonsense in here. Noticed we didnt need this since we have a setup.